apisix/plugins/hmac-auth.lua

```diff
@@ -52,16 +58,19 @@ local schema = {
                 clock_skew = {
                     type = "integer",
                     default = 300
-                }
+                },
+                signed_headers = {
+                    type = "array",
+                    items = {
+                        type = "string",
+                        minLength = 1,
+                        maxLength = 20,
```

Comment:
The maxLength of 20 is not enough, how about 50?

Reply:
ok